gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -123,7 +127,7 @@ object ConverterUtils extends Logging {
   // TODO: This is used only by `BasicScanExecTransformer`,
   //  perhaps we can remove this in the future and use `withExprId` version consistently.
   def collectAttributeNamesWithoutExprId(attributes: Seq[Attribute]): JList[String] = {
-    collectAttributeNamesDFS(attributes)(genColumnNameWithoutExprId)
+    collectAttributeNamesDFS(attributes)(getAttributeName)
```

Comment:
Seems `getAttributeName` function can be replaced with `normalizeColName(attr.name)` directly, thanks.

Reply:
ok.